cmd/legacy_main.go

```diff
@@ -87,13 +89,24 @@ func registerTerminatingSignalHandler(mountPoint string, c *cfg.Config) {
 
 func getUserAgent(appName string, config string) string {
 	gcsfuseMetadataImageType := os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")
+	mountId := getUniqueMountID()
```

Comment:
I feel, we should make caller's responsibility to pass the unique ID. Other wise, different invocation with the same parameter will return different user-agent string.